packages/react-composites/src/composites/common/PeoplePaneContent.tsx

```diff
@@ -152,8 +147,12 @@ const createDefaultContextualMenuItems = (
   strings: CallWithChatCompositeStrings | /* @conditional-compile-remove(one-to-n-calling) */ CallCompositeStrings,
   onRemoveParticipant: (userId: string) => Promise<void>,
   localParticipantUserId?: string,
-  disableRemovebutton?: boolean
+  disableRemoveButton?: boolean
 ): IContextualMenuItem[] => {
+  let disabled = !participant.isRemovable;
```

Comment:
nit: This can simply be `const disabled = !participant.isRemovable || disableRemoveButton`

Reply:
`disableRemoveButton` is an optional argument and might be undefined, just added a check before updating the value of `disabled`